src/components/SettlementButton.tsx

```diff
@@ -197,7 +197,7 @@ function SettlementButton({
         }
 
         if (iouPaymentType === CONST.IOU.REPORT_ACTION_TYPE.APPROVE) {
-            IOU.approveMoneyRequest(iouReport);
+            IOU.approveMoneyRequest(iouReport ?? {});
```

Comment:
shouldn't this be checked in the if statement? I mean, are we sure we can pass an empty object into that fn if it's approved?